saylua/provisioner.py

```diff
@@ -119,6 +119,8 @@ def purge(absolutely_sure_about_this=False):
 
 
 def setup():
+    db.create_all()
```

Comment:
I don't approve of this change, I think we should have convened the council and galactic board on this one.